aws/table_aws_ssm_association.go

```diff
@@ -151,7 +151,15 @@ func tableAwsSSMAssociation(_ context.Context) *plugin.Table {
 			{
 				Name:        "targets",
 				Description: "A cron expression that specifies a schedule when the association runs.",
+				Hydrate:     getAwsSSMAssociation,
```

Comment:
Description should be `The instances targeted by the request to create an association.`